test/Constraints/rdar37291371.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-typecheck-verify-swift
+
+extension Collection where Element: Numeric {
+    var v: Element {
+        return self.reduce(0, +)
+    }
+}
```

Comment:
Nit: The indentation here is 4 spaces whereas the rest of the file is 2.

Reply:
Ah, I will fix this momentarily...